documentation/asciidoc/computers/configuration/access-point-routed.adoc

```diff
@@ -152,7 +152,7 @@ address=/gw.wlan/192.168.4.1
 
 The Raspberry Pi will deliver IP addresses between `192.168.4.2` and `192.168.4.20`, with a lease time of 24 hours, to wireless DHCP clients. You should be able to reach the Raspberry Pi under the name `gw.wlan` from wireless clients.
 
-NOTE: In the personal IP network range ,There are three types: `10.0.0.0 to 10.255.255.255`,`172.16.0.0 to 172.31.255.255` and `192.168.0.0 to 192.168.255.255`. Adjust the range accordingly and use it.
+NOTE: In the personal IP network range, There are three types: `10.0.0.0 to 10.255.255.255`,`172.16.0.0 to 172.31.255.255` and `192.168.0.0 to 192.168.255.255`. Adjust the range accordingly and use it.
```

Comment:
Shouldn't be a comma there in the first place, and there shouldn't be a capital letter after a comma. But putting both of those aside, that isn't even a sentence. Maybe this,

```
NOTE: There are three IP address blocks set aside for private networks. There is the Class A block, from `10.0.0.0` to `10.255.255.255`, a Class B block from `172.16.0.0` to `172.31.255.255`, and probably the most frequently used, a Class C block from `192.168.0.0` to 192.168.255.255`.
```

But that said, I'm not sure how relevant or useful the `NOTE` is anyway? 🤷‍♂️

Reply:
@aallan 
IMHO, I think it's good if it's a guide for the beginner tutorial.

Reply:
Fair

Reply:
@aallan 
Thank you for reflecting it :smile: